doc/source/rest.j2

```diff
@@ -602,7 +602,16 @@ least one of the time series is returned.
 
 {{ scenarios['get-aggregates-by-metric-ids-fill']['doc'] }}
 
+It's also possible to do that aggregation on |metrics| linked to |resources|.
+In order to select these |resources|, the following endpoint accepts a query
+such as the one described in `Searching for resources`_.
```

Comment:
I'm thinking we'll have to rewrite this as a string like we did for operations, e.g., `(eq "server_group" "my_autoscaling_group")`

Reply:
That looks a good idea !